client/src/components/ElementEditor/tests/Header-test.js

```diff
@@ -1,307 +1,217 @@
+  })}
+  />);
+  expect(container.querySelector('.test-element-actions')).not.toBeNull();
+});
 
-      const versionedState = wrapper.find('.element-editor-header__version-state');
-      expect(versionedState.prop('title')).toContain('not been published');
-      expect(versionedState.hasClass('element-editor-header__version-state--draft')).toBe(true);
-    });
+test('Header should render an ElementActions component when the element is not expandable', () => {
+  const { container } = render(<Header {...makeProps({
+    expandable: false,
+  })}
+  />);
+  expect(container.querySelector('.test-element-actions')).not.toBeNull();
+});
 
-    it('identifies modified versions', () => {
-      element.isPublished = true;
-      element.isLiveVersion = false;
-      const wrapper = shallow(
-        <Header
-          areaId={1}
-          type={type}
-          ElementActionsComponent={ElementActionsComponent}
-          element={element}
-          connectDragSource={content => content}
-          connectDragPreview={content => content}
-          onDragEnd={() => {}}
-        />
-      );
+test('Header should render an ElementActions component even when the element is broken', () => {
+  const { container } = render(<Header {...makeProps({
+    type: {
+      broken: true,
+      obsoleteClassName: 'RemovedClass'
+    }
+  })}
+  />);
+  expect(container.querySelector('.test-element-actions')).not.toBeNull();
+});
 
-      const versionedState = wrapper.find('.element-editor-header__version-state');
-      expect(versionedState.prop('title')).toContain('has unpublished changes');
-      expect(versionedState.hasClass('element-editor-header__version-state--modified')).toBe(true);
-    });
+test('Header should render a versioned state message when the element is not published', () => {
+  const { container } = render(<Header {...makeProps({
+    element: {
+      id: '14',
+      isPublished: false,
+      liveVersion: false
+    }
+  })}
+  />);
+  expect(container.querySelector('.element-editor-header__version-state--draft').getAttribute('title')).toContain('not been published');
+});
 
-    it('ignores live versions', () => {
-      element.isPublished = true;
-      element.isLiveVersion = true;
-      const wrapper = shallow(
-        <Header
-          areaId={1}
-          type={type}
-          ElementActionsComponent={ElementActionsComponent}
-          element={element}
-          connectDragSource={content => content}
-          connectDragPreview={content => content}
-          onDragEnd={() => {}}
-        />
-      );
+test('Header should render a versioned state message when the element is modified', () => {
+  const { container } = render(<Header {...makeProps({
+    element: {
+      id: '14',
+      isPublished: true,
+      isLiveVersion: false
+    }
+  })}
+  />);
+  expect(container.querySelector('.element-editor-header__version-state--modified').getAttribute('title')).toContain('has unpublished changes');
```

Comment:
```suggestion
  expect(container.querySelector('.element-editor-header__version-state.element-editor-header__version-state--modified').getAttribute('title')).toContain('has unpublished changes');
```
The `.element-editor-header__version-state` class is relevant, because that's what we're checking _isn't_ rendered in the below test.
This applies for the above test as well.

Reply:
Not doing